doc/ref_model/chapters/chapter08-annex.md

```diff
@@ -139,3 +140,511 @@ Example, or Reference, Functest Status Reporting Artifacts:
+
+<p><strong>Targeted VNF Classes/Families </strong>for baseline measurements are described in <a href=https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#2.2>chapter 2.2</a>.
+
+<a name="x.x.x"></a>
+### Best Practices.
+<ul>
+<li>Standardized test methodology / flow, Test Plan, and Test Case Suites</li>
+<li>Integration with Dovetail and OVP flow (code, docs, cert criteria, etc.)</li>
+<li>Leveraging ONAP Network and Service Models, with identified VNF-specific parameters</li>
+<li>Standardized certification criteria</li>
+<li>Define CNTT ref arch as scenarios, and have all test cases for the ref arch be involved in OVP (could also be addressed in OVP as CNTT test)</li>
+<li>Add test cases from operators, which operators already tested in their environment</li>
+</ul>
+
+
+
+<a name="x.x.x"></a>
+### System Under Test (SUT) Pre-reqs
+1. Test Specifications – per infra profile, specifications, and features/capabilities
+2. Naming conventions
+3. NFVi profiles and Flavours
+4. Test User Guide
+5. Scenarnio Descriptor File (SDF)
+
+
+<a name="x.x.x"></a>
+### Test Categories
+<ol>
+<li>On-Boarding (ONAP MANO, NFVi Agnostic)</li>
+<li>Instantiation, Deletion, Recent Change (ONAP MANO, NFVi Agnostic)</li>
+<li>VNF Functional Testing</li>
+<li>Security Testing</li>
+<li>Charging / Revenue Assurance Verification</li>
+<li>MicroServices Support</li>
+<li>Closed Loop Testing</li>
+<li>VNF Coexistence (ETSI NFV-TST001 &ldquo;Noisy Neighbor&rdquo;)</li>
+<li>VNF Interactions with Extended NFVi Topology</li>
+<li>VNF Interactions with Complex NFVi (Akraino)</li>
+<li>Scalability Testing</li>
+<li>HA Testing</li>
+<li>Fault Recovery Testing</li>
+<li>PM/KPI/Service Assurance Testing</li>
+</ol>
+
+<a name="x.x.x"></a>
+### Test Harness(es)
+<ol>
+<li>Standardized test methodology / flow</li>
+<li>Working test bed reference design</li>
+<li>Standardized Cloud-based facilities (storage, IAM, etc.)</li>
+<li>Standard ONAP Network and Service Models, with identified VNF-specific parameters</li>
+<li>Robot library to enable Data-Driven testing of On-Boarding, Instantiation, etc.</li>
+<li>Sample VNF, CSAR and Robot Test Cases</li>
+<li>Standardized base Test Plan and Test Case suite</li>
+<li>Standardized certification criteria</li>
+<li>Integration with Dovetail and OVP flow (code, docs, cert criteria, etc.)</li>
+<li>Real, usable documentation</li>
+<li>&ldquo;Just add Water&rdquo; deployment vehicle</li>
+</ol>
+
+<a name="x.x.x"></a>
+### Test Tools
+1. Shaker:  https://pyshaker.readthedocs.io/en/latest/ (The distributed data-plane testing tool built for OpenStack)
+2. Sonubuoy: https://sonobuoy.io/ It is a diagnostic tool that makes it easier to understand the state of a Kubernetes cluster by running a set of plugins (including Kubernetes conformance tests) in an accessible and non-destructive manner.
+
+<a name="x.x.x"></a>
+### Scenario Descriptor File (SDF)
```

Comment:
Suggest removal as those have been moved to chapter 8.

Reply:
Agree. Removed. will mark as resolved.